easybuild/easyconfigs/m/magma/make.inc

```diff
@@ -0,0 +1,49 @@
+#//////////////////////////////////////////////////////////////////////////////
```

Comment:
can you point me out to those lines on the log I've sent? Browsing the whole debug log for non-eb experts may take some time :P